### DISCUSSION TITLE
S200102 - Hawaii Hydrojet Pro

### DISCUSSION THREAD

**Discussion by bluemryld** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#349] [Q&A] [answered]
Hi,

I have a Hawaii Hydrojet Pro with a S200102 Square Pump.

I have ordered and built a V2 PCB. 

When powered up the display plays the tones and after a short time displays what i believe is the version number. The panel then partially lights (C F and power button)

The web app is able to be opened however the toggles do not function they immediately turn off again without any visible result, with the exception of the C/F toggle which causes the set and current temperature to cycle through a set of values.

I have used both a LLC and jumper wires and get them same result with each. I have checked by soldering and connections and all seems fine.

Is there a way to get extra debug data so i can see whats going on? Can i put the board in passive passthrough with capture or something to verify the connections?

What can i check? Is this pump compatible?

**Comment by SigmaPic** [comment, marked as answer]
Check issue #330. We just success to make it works for your pump model.

You shall use another pinout with PCB_V2B.

Everything is detailed in #330.

**Reply by bluemryld** [reply]
Awesome that seems to work, even the hydrojets.

I wired the spaghetti way and changed the pcb to v2b using the dev branch and it all seems good.

I am happy to have a go with the v2b board once its ready to try.

Next job is home assistant/node red :)

**Reply by henryfabu** [reply]
Hi, I have a Maldives spa with s200102 pump.
I have been reading all the project instructions, and the help forums, but I think I don't hace the skills to succesfully do this project.
Do you think it is possible that someone from here, who knows about the required configuration for my spa and pump model, could make the device for me? Of course I would pay for the conponents and the work.

Thank you very much and I am sorry if it is not the correct place for my post.
Best regards

**Reply by ejlersen64** [reply]
Hello.
I have the same problem, did you ever get an answer.

**Comment by skyluker** [comment]
I have used PCB_V1 with standard level shifter (txe) with 3x 580ohms on pins 3,4,5 and it works.

**Reply by Deefjes** [reply]
How did you do the wiring ?

1 on 1 ?

(I'm struggling with the Hawaii Hydrojet Pro 2021+. Tested yesterday with the PCB_V2B definition and wiring, but got like white noise from the DSP)

Would like to try your setup with 3x580 Ohm, TXE and PCB_V1. I've got a V1 PCB board.

**Reply by SigmaPic** [reply]
> How did you do the wiring ?
> 
> 1 on 1 ?
> 
> (I'm struggling with the Hawaii Hydrojet Pro 2021+. Tested yesterday with the PCB_V2B definition and wiring, but got like white noise from the DSP)
> 
> Would like to try your setup with 3x580 Ohm, TXE and PCB_V1. I've got a V1 PCB board.

What LLC do you use ?
You can try the hardware test by browsing the hwtestinfo.html on your esp.
You have to connect both connectors together out of the pump and run the hw test.

**Reply by Deefjes** [reply]
LLC = TXS0108E.

But I've also ordered the MOSFET one. Which will arive today hopefully.

Ok. So for the hardware test, just connect the connectors to the V1 PCB and run the hwtestinfo.html ?
So CIO to H1 and DISP to H2 ?

**Reply by Deefjes** [reply]
Sorry. I'm stupid. hwtestinfo.html has pictures of what to do. Derpy Derp

**Reply by SigmaPic** [reply]
> LLC = TXS0108E.
> 
> But I've also ordered the MOSFET one. Which will arive today hopefully.
> 
> Ok. So for the hardware test, just connect the connectors to the V1 PCB and run the hwtestinfo.html ?
> So CIO to H1 and DISP to H2 ?
> 

We confirmed that it works with the MOSFET version of the LLC.

You remove the PCB from the pump (ok n the desk) and you connect both connectors together

**Comment by Deefjes** [comment]
Did the hardware test. Seems like I've got the cabling wrong.

Hardware:
- PCB V1
- LLC: TXS0108E
- NodeMCU v3

Software/code settings:
PCB_V2B _(will try with setting to PCB_V1, with and without 580 Ohm R's)_

Hardwaretest results:
DSP input pin 2 500 errors of 1000
DSP input pin 3 300 errors of 1000
DSP input pin 4 390 errors of 1000
CIO input pin 2 500 errors of 1000
CIO input pin 3 0 errors of 1000
CIO input pin 4 500 errors of 1000

**Reply by Deefjes** [reply]
Jup. Had the cabling wrong:

**Reply by visualapproach** [reply]
Your best bet is to use PCB_V2B and connect the wires accordingly. It will be a braid but that's okay as long as you connect them right

**Comment by jozper** [comment]
Hi, I have Hawaii S200102, software uploaded to nodemcu, but it Hardware config Im not sure, what to select. In the manual is, that Hawai S200102 is Maldives2021, but in menu of hw config is: pre2021, 2021, 54149E, 54173 etc...

**Reply by visualapproach** [reply]
Yes, the factory naming convention is messy. I guess you have a 6-wire device and should try 2021 with matching features (hydrojets)

**Reply by jozper** [reply]
thank you, I will try
